frontend/src/lib/components/UniversalSearch/UniversalSearch.scss

```diff
@@ -16,25 +17,20 @@
     @media screen and (min-width: $sm) {
         display: flex;
     }
+
+    .hidden {
+        display: none;
+        opacity: '0';
```

Comment:
Opacity is a number, it doesn't work as a string. But this `.hidden` class doesn't seem really useful overall. Why not just make the component being rendered conditional directly in JS, since this has exactly the same effect?

Reply:
... that makes a lot of sense 🤦